Mathlib/LinearAlgebra/TensorProduct.lean

```diff
@@ -681,6 +681,9 @@ theorem comm_symm_tmul (m : M) (n : N) : (TensorProduct.comm R M N).symm (n ⊗
   rfl
 #align tensor_product.comm_symm_tmul TensorProduct.comm_symm_tmul
 
+lemma lift_comp_comm_eq  {f : M →ₗ[R] N →ₗ[R] P} :
+    (lift f) ∘ₗ (TensorProduct.comm R N M) = lift f.flip := by
+  exact ext rfl
```

Comment:
Do you need the `by exact`?
```suggestion
lemma lift_comp_comm_eq (f : M →ₗ[R] N →ₗ[R] P) :
    lift f ∘ₗ TensorProduct.comm R N M = lift f.flip :=
  ext rfl
```
(also `f` should be explicit)

Reply:
No, I don't need it. I followed your suggestion and removed a few other "by exact"